Tests/govuk_ios/govuk_ios_snapshot_tests/Helpers/SnapshotTestCase.swift

```diff
@@ -1,13 +1,18 @@
 import UIKit
 
 import iOSSnapshotTestCase
+import Factory
+import Lottie
 
 @testable import govuk_ios
 
 class SnapshotTestCase: FBSnapshotTestCase {
 
     override func setUp() {
         super.setUp()
+        Container.shared.lottieConfiguration.register {
+            LottieConfiguration(renderingEngine: .mainThread)
+        }
 //        self.recordMode = true
```

Comment:
Commented code

Reply:
This isn't part of this PR. It was added as part of the introduction of Snapshot testing.
It is helpful if you need to retake all snapshot references for some reason (new iPhone). We can take it out and add some documentation to the README about it if preferred, but I will do it in another PR